--- a/verl/utils/reward_score/__init__.py
+++ b/verl/utils/reward_score/__init__.py
@@ -87,8 +87,8 @@ def default_compute_score(data_source, solution_str, ground_truth, extra_info=No

         # res = math_dapo.compute_score(solution_str, ground_truth)

-        from . import math_verify
-        res = math_verify.compute_score(solution_str, ground_truth)
+        from . import math
+        res = math.compute_score(solution_str, ground_truth)

         # from . import deepmath
         # res = deepmath.compute_score(solution_str, ground_truth)
